### DISCUSSION TITLE
Consider documenting that `indent.scope` is the way to show indent guides for the current scope only

### DISCUSSION THREAD

**Discussion by nadiamoe** [folke/snacks.nvim#332] [Ideas]
I wanted this behavior:
![image](https://github.com/user-attachments/assets/0a2ecb04-7a6c-4045-9bdf-21b1752430dd)

Took me a good while to figure out that I did not want:
```lua
{
  "folke/snacks.nvim",
  opts = {
    indent = {
      indent = {
        current_only = true,	
      },
    },
  },
}
```

But rater:
```lua
{
  "folke/snacks.nvim",
  opts = {
    indent = {
      indent = {
        enabled = false,
      },
    },
  },
}
```

As it is `indent.scope`, and not `indent.indent` the one taking care of _that_ indent line.

**Comment by SirWrexes** [comment]
Thank you my saviour for making my Nix files not look like this hellish sideways roller coaster anymore. :pray: 
![image](https://github.com/user-attachments/assets/00a6fefb-b7d5-4350-a2a2-0f04d360c288)
